src/qredisclient/connection.cpp

```diff
@@ -332,6 +332,11 @@ RedisClient::DatabaseList RedisClient::Connection::getKeyspaceInfo() {
 
 void RedisClient::Connection::refreshServerInfo() {
   Response infoResult = internalCommandSync({"INFO", "ALL"});
+  if (infoResult.value().toByteArray().startsWith("NOPERM")) {
```

Comment:
Please add `isPermissionError` method to `Response` class (example https://github.com/uglide/qredisclient/blob/master/src/qredisclient/response.cpp#L66)